hw4/01-routing.js

```diff
@@ -40,8 +40,46 @@ app.get('/', (req, res) => {
   res.end();
 });
 
-app.get('/welcome', (req, res) => {});
+app.get('/welcome', (req, res) => {
+  res.writeHead(200, { 'Content-Type': 'text/html' });
+  res.write(`<h1>Welcome</h1>`);
```

Comment:
You can look into `res.send()` which is used in ExpressJS. You can replace line 45 and 46 with just a single res.send() call.

_https://stackoverflow.com/questions/44692048/what-is-the-difference-between-res-send-and-res-write-in-express_

Reply:
Right, This is a great input I will make this change.